programs/staking/src/lib.rs

```diff
@@ -189,6 +189,27 @@ mod staking {
 
         Ok(())
     }
+
+    pub fn change_funder(ctx: Context<FunderChange>, funder: Pubkey) -> Result<()> {
+        let vault = &mut ctx.accounts.vault;
+        if funder == vault.funder.key() {
+            return Err(VaultError::FunderAlreadyAuthorized.into());
+        }
+        vault.funder = funder;
+        Ok(())
+    }
+
+    pub fn deauthorized_funder(ctx: Context<FunderChange>, funder: Pubkey) -> Result<()> {
```

Comment:
I think we dont need this function, If we want to deauthorized_funder, we just change_funder to Pubkey::default()

Reply:
resolved